core/clingutils/src/TClingUtils.cxx

```diff
@@ -682,12 +682,7 @@ ROOT::TMetaUtils::TNormalizedCtxtImpl::TNormalizedCtxtImpl(const cling::LookupHe
    keepTypedef(lh, "Long64_t", true);
    keepTypedef(lh, "ULong64_t", true);
 
-   clang::QualType toSkip = lh.findType("string", cling::LookupHelper::WithDiagnostics);
```

Comment:
As far as I know since 97119a8795f76d9693b24658982c59f25f0336d0 (@Axel-Naumann), this is not duplicate code. We need to record the TypedefDecl for both "string" and "std::string" and the code using this information intentional does not desugar.

Reply:
Ok, I think I overlooked the code. I will fix this. I was trying to make this code work without having `using namespace std;` as it currently crashes.